firmware/arduino/chessboard/chessboard.ino

```diff
@@ -18,6 +18,7 @@ volatile char messageBuffer2[INCOMING_MESSAGE_LENGTH];
 volatile char * tempCharPtr;
 volatile char * rxBufferPtr = messageBuffer1;
 volatile char * receivedMessagePtr = messageBuffer2;
+volatile char sentMessage[3];
```

Comment:
Please add a comment above explaining what each field in `sentMessage` is. Something liek
```c
// sentMessage[] holds currentState, errorCode, and moveCount (in that order)
```